torchio/data/image.py

```diff
@@ -132,3 +138,23 @@ def load(self, check_nans: bool = True) -> Tuple[torch.Tensor, np.ndarray]:
         if check_nans and torch.isnan(tensor).any():
             warnings.warn(f'NaNs found in file "{self.path}"')
         return tensor, affine
+
+    def is_2d(self) -> bool:
+        return self.shape[-3] == 1
+
+    def numpy(self) -> np.ndarray:
+        return self[DATA].numpy()
+
+    def as_sitk(self) -> sitk.Image:
+        return nib_to_sitk(self.data, self.affine)
+
+    def get_center(self, lps: bool = False) -> TypeTripletFloat:
+        """Get image center in RAS (default) or LPS coordinates."""
+        image = self.as_sitk()
+        size = np.array(image.GetSize())
+        center_index = (size - 1) / 2
+        l, p, s = image.TransformContinuousIndexToPhysicalPoint(center_index)
+        if lps:
+            return (l, p, s)
+        else:
+            return (-l, -p, s)
```

Comment:
I may be a little bit confused, here: I do not understand why you should take care of RAS or LPS ?
My understanding is that one needs to define a point in world coordinate (RAS or LPS are for the anatomical coordinate system as explain here https://www.slicer.org/wiki/Coordinate_systems
But I think your right because sitk uses a LPS coordinate system ... ?

Reply:
ITK (and DICOM) assumes the first two dimensions to go left and posterior, whereas NIfTI likes right and anterior.